experiments/list5/README.md

```diff
@@ -0,0 +1,123 @@
+# LisT5: FEVER Pipeline with T5
```

Comment:
Can we add a pointer to our paper from ACL Anthology as well as a very short description?